scheduler/src/cook/kubernetes/api.clj

```diff
@@ -390,8 +441,11 @@
 
     ; container
     (.setName container cook-container-name-for-job)
-    (.setCommand container
-                 ["/bin/sh" "-c" (:value command)])
+    (if wrapper-script
+      (.setCommand container
+                   ["/bin/sh" "-c" (str wrapper-script-workdir "/" wrapper-script " '" (str/escape (:value command) {\' "'\\''"}) "'")])
```

Comment:
Why do we need `str/escape` here?

Why don't we just assume that the wrapper (launcher?) script is executable, then this becomes:

```clojure
[(str wrapper-script-workdir "/" wrapper-script) (:value command)]
```

Reply:
will call it custom shell and remove the escaping